dll/win32/shell32/shellrecyclebin/recyclebin.c

```diff
@@ -3,7 +3,7 @@
  * LICENSE:     GPL v2 - See COPYING in the top level directory
  * FILE:        lib/recyclebin/recyclebin.c
  * PURPOSE:     Public interface
- * PROGRAMMERS: Copyright 2006-2007 Herv� Poussineau (hpoussin@reactos.org)
+ * PROGRAMMERS: Copyright 2006-2007 Hervé Poussineau (hpoussin@reactos.org)
```

Comment:
Why is this line touched? Intended?

Reply:
Exactly the same as the next one

Reply:
Since this file is not otherwise modified for this PR, please revert this change (by just reverting the whole recyclebin.c)
It will make sense only in other PRs that actually implement stuff in this recyclebin.c , to have this change.